internal/schema/private_endpoints.go

```diff
@@ -0,0 +1,60 @@
+package schema
+
+import (
+	"fmt"
+
+	"github.com/hashicorp/terraform-plugin-framework/types"
+	"github.com/hashicorp/terraform-plugin-framework/types/basetypes"
+
+	"github.com/couchbasecloud/terraform-provider-couchbase-capella/internal/errors"
+)
+
+// PrivateEndpoint represents a private endpoint resource.
+type PrivateEndpoint struct {
+	// EndpointId is the id of the bucket for which the collection needs to be created.
```

Comment:
Change the docstrings everywhere
```suggestion
	// EndpointId is the id of the bucket for which the private endpoint needs to be created.
```

Reply:
addressed in 9a0365f